lib/ssl/doc/src/ssl.xml

```diff
@@ -299,6 +303,15 @@
       and to restrict their usage when using a cipher suite supporting them.
       </p></item>
 
+      <tag><c>{supported_groups, [group()]}</c></tag>
+      <item><p> Used only in TLS 1.3 and obsoletes the eccs option. It indicates the named
+      groups which the client supports for key exchange, ordered from most preferred to least
```

Comment:
maybe say groups (Diffie-Hellman and Elliptic-Curve groups)

Reply:
Will fix.